packages/algorand-js/src/interpreter/opcode-list.ts

```diff
@@ -80,3 +81,37 @@ export class Arg extends Op {
     stack.push(a);
   }
 }
+
+// load block of byte-array constants
+export class Bytecblock extends Op {
+  readonly bytecblock: Uint8Array[];
+  readonly interpreter: Interpreter;
+
+  constructor (interpreter: Interpreter, bytecblock: Uint8Array[]) {
+    super();
+    this.interpreter = interpreter;
+    this.bytecblock = bytecblock;
```

Comment:
we don't need to store `bytecblock`. It's already in the interpreter.

Reply:
Sorry, i don't follow. `this.bytecblock` is storing the `bytecblock` provided explicitly to the `new Bytecblock(..)` constructor. This is from what we are updating the interpreter `bytecblock` with.

Reply:
You are right. sorry.